### DISCUSSION TITLE
Does XRDP work well when full-screen DPI/resolution changes?

### DISCUSSION THREAD

**Discussion by HubKing** [neutrinolabs/xrdp#2290] [Q&A] [answered]
I am not sure if this a bug of XRDP or not so before raising an issue, I want to make it sure. I have one monitor (1080p, 125% DPI) and another monitor (4K, 200% DPI). I connect to a Linux XRDP server using Windows' in-built client with the resolution set to "Full Screen". When it is connected, I place the mouse over the top edge until the blue bar appears. Then I drag it to the top of the other monitor so that it will be full-screen on that monitor. If I do that, the client shows the following error and gets closed. I do not think I experienced this when connecting to a Windows RDP server.

![image](https://user-images.githubusercontent.com/10900489/173796634-47b51c26-5640-4c72-b20f-1f3be35cbedc.png)

**Comment by matt335672** [comment, marked as answer]
This is probably #1928. It's not fixed in an official build, but we're aware of it.
